application/controllers/Treebank.php

```diff
@@ -133,18 +123,26 @@ public function delete($treebank_id)
         // Delete the treebank from the database
         $this->treebank_model->delete_treebank($treebank_id);
 
+        // Delete files
+        if (file_exists(UPLOAD_DIR.$treebank->filename)) {
+            unlink(UPLOAD_DIR.$treebank->filename);
+        }
+        $file_path_without_extension = UPLOAD_DIR.substr($treebank->filename, 0, -4);
```

Comment:
Is there a specific reason for doing this with a substring and not any of the methods mentioned [here](https://stackoverflow.com/questions/2183486/php-get-file-name-without-file-extension)?

Reply:
Good point! I did it this way because this was how it was already done. But using `pathinfo` instead is more sensible, so I've modified it to use that instead now.